src/content/en/updates/2016/10/devtools-digest.md

```diff
@@ -0,0 +1,79 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: New Console features, updates on the context selector bug, and the new UC Browser user agent.
+
+{# wf_updated_on: 2016-10-18 #}
+{# wf_published_on: 2016-10-18 #}
+{# wf_tags: devtools #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+
+# DevTools Digest, October 2016 {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+Hey hey, [Kayce](https://twitter.com/kaycebasques) here, back again for
+another digest to update you on what's new in DevTools since last month.
+
+[Crbug]: https://bugs.chromium.org/p/chromium/issues/list?can=1&q=component%3APlatform%3EDevTools+-status%3AUnconfirmed+-status%3AUntriaged++-status%3AAvailable+-status%3AAssigned+-status%3AStarted+-status%3AExternalDependency+-status%3ADuplicate+-status%3AWontFix+-status%3AArchived+&sort=-modified&colspec=ID+Pri+M+Stars+ReleaseBlock+Component+Status+Owner+Summary+OS+Modified&x=m&y=releaseblock&cells=ids
+
+## New features in the Console {: #console }
+
+In Chrome 56, which is currently Canary, the DevTools Console is powered by
+[CodeMirror](https://codemirror.net/). This enables a whole bunch of new
+features, like:
+
+**Syntax highlighting as you type**. Previously, DevTools could only
+highlight syntax after a code block had been evaluated.
+
+**Matching bracket highlighting**. Because no one got time for those annoying
+`Uncaught SyntaxError: Unexpected token {` errors.
+
+**Smart return**. When you type out a multi-line code block, DevTools
+now knows whether to create a new line or execute your code each time you
+press <kbd>Enter</kbd>. You can still force DevTools to create a new line by
+holding <kbd>Shift</kbd> before you press <kbd>Enter</kbd>.
```

Comment:
Are you saying I no longer need to write functions on a single line? Might make that clearer. You might compare this to command line interpreters in other languages like Python for instance. (I think Ruby might also do that. Ask around.)